src/App.tsx

```diff
@@ -1,173 +1,46 @@
-import { Loader } from './components/Loader';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { Navigation } from './components/Navigation';
 
 import './App.scss';
-
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
-        </div>
-      </div>
-    </nav>
-
-    <main className="section">
-      <div className="container">
-        <h1 className="title">Home Page</h1>
-        <h1 className="title">People Page</h1>
-        <h1 className="title">Page not found</h1>
-
-        <div className="block">
-          <div className="box table-container">
-            <Loader />
-
-            <p data-cy="peopleLoadingError" className="has-text-danger">
-              Something went wrong
-            </p>
-
-            <p data-cy="noPeopleMessage">
-              There are no people on the server
-            </p>
-
-            <table
-              data-cy="peopleTable"
-              className="table is-striped is-hoverable is-narrow is-fullwidth"
+import { Table } from './components/Table';
+
+export const App = () => {
+  return (
+    <div data-cy="app">
+      <Navigation />
+
+      <main className="section">
+        <div className="container">
+          <Routes>
+            <Route
+              path="*"
+              element={<h1 className="title">Page not found</h1>}
+            />
```

Comment:
If I'm not mistaken, there is a router with "*" in the video lectures. But ok, I didn't know that I'll use it!

Reply:
> "*" route should always come last in the list of routes, otherwise other routes will never work out.

I don't pass the tests when I put a route with "*" at the end